lms/djangoapps/learner_dashboard/models.py

```diff
@@ -0,0 +1,24 @@
+"""Models for Learner's Dashboard"""
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class LearnerDashboardConfiguration(ConfigurationModel):
```

Comment:
We talked about this briefly offline, but I'd like to continue our conversation here. I think it would be ideal if we only had one config model responsible for controlling program-related functionality in edx-platform. ProgramsApiConfig is unfortunately named, but it's shaping up to be that one config model for us. In spite of the name, it feels like the right place for the configuration you need to add in this PR.